src/field_theory/abel_ruffini.lean

```diff
@@ -372,6 +372,30 @@ begin
   { exact λ α n, induction3 },
 end
 
+lemma is_solvable_contrapositive {α : E} {q : polynomial F} (q_irred : irreducible q)
+  (q_aeval : aeval α q = 0) (q_solvable : ¬ _root_.is_solvable q.gal) :
+  ¬ is_solvable_by_rad F α :=
```

Comment:
Do you want to state this with two `\not`s?

Reply:
The specific reason is that this is the statement that will be used in practice, but I suppose it might make sense to swap `q_solvable` with the goal, and remove the nots.

Reply:
Can you then also provide an `iff` version, for completeness?

Reply:
Not yet. We don't have the converse of abel-ruffini yet, and I think the converse also requires a separable assumption.

Reply:
> The specific reason is that this is the statement that will be used in practice, but I suppose it might make sense to swap `q_solvable` with the goal, and remove the nots.

If you remove the `nots` then you get this version by using `mt`. So I would prefer removing the nots.